bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,437 @@
+| shelly2dimmer      | Shelly Dimmer                                          | fully supported                                          |
+| shellyht           | Shelly Sensor (temp+humidity)                          | fully supported                                          |
+| shellyplugs        | Shelly Plug-S                                          | fully supported                                          |
+| shellyplug         | Shelly Plug                                            | fully supported                                          |
+| shellyrgbw2        | Shelly RGB Controller                                  | fully supported                                          |
+| shellybulb         | Shelly Bulb in Color or WHite Mode                     | fully supported                                          |
+| shellysense        | Shelly Motion and IR Controller                        | fully supported                                          |
+| shelly4pro         | Shelly 4x Relay Switch                                 | fully supported                                          |
+| shellysmoke        | Shelly Sensor (temp+humidity)                          | should get discovered, but no special handling yet       |
+
+
+## Firmware
+
+To utilize all features the binding requires firmware version 1.5.0 or newer.
+This should be available for all devices.
+Older versions work in general, but have impacts to functionality (e.g. no events for battery powered devices).
+
+The binding displays a WARNING if the firmware is older.
+It also informs you when an update is available.
+Use the device's web ui or the Shelly App to perform the update.
+
+
+
+## Discovery
+
+The binding uses mDNS to discovery the Shelly devices.
+They periodically announce their presence, which can be used by the binding to find them on the local network and fetch their IP address.
+The binding will then use the Shelly REST api (http) to discover device capabilities, read status and control the device.
+In addition event callbacks will be used to support battery powered devices.
+The binding also support the CoIoT Spec, which is based on Coap. 
+
+### Password protected devices
+
+The Shelly Apps allow to protect device configuration by userid+password, which is  supported by the binding.
+If you are using password protected Shelly devices you need to configure userid and password, otherwise a thing is created (shelly-protected) to indicate that a password protected device was discovered (rthe dicovered information includes the device type and IP address).
+
+Change the binding configuration and re-discover the device.
+
+1. Global Default: Go to PaperUI:Configuration:Addons:Shelly Binding and edit the configuration.
+Those will be used when now settings are given on the thing level.
+2. Edit the thing configuration. 
+
+Important: The IP address shouldn't change after the device is added as a new thing in openHAB. This could be achieved by
```

Comment:
So this means that you identify devices by their IP address? Don't they have any kind of unique identifier that they send with their mDNS announcement and which could serve as the primary id so that changing IP addressed could be covered?